music/src/core/player.ts

```diff
@@ -78,13 +79,27 @@ export abstract class BasePlayer {
    *   @param callbackObject An optional BasePlayerCallback, specifies an
    *     object that contains run() and stop() methods to invode during
    *     playback.
+   *   @param bpm An optional number specifying the starting BPM for playback.
    */
-  constructor(playClick = false, callbackObject?: BasePlayerCallback) {
+  constructor(playClick = false, callbackObject?: BasePlayerCallback,
+              bpm = 60) {
```

Comment:
I don't think we actually need to set an initial BPM in the constructor since it is set in `start()` now.

Reply:
but it's only allowed for quantized sequences, so for quantized sequences you could only do it via the constructor or via setBPM. i can remove the constructor one if you feel it would be simpler to just do it via setBPM, or i could also remove the restriction in start().